### PR TITLE
i18n(id): Update customization.mdx and configuration.mdx translation

### PR DESCRIPTION
**Pull request by iyyafi**
#### Description
- Update ID translation guides for customization.mdx file
- Update ID translation reference for configuration.mdx file

### CODE REVIEW

File: docs/src/content/docs/id/guides/customization.mdx [hunk 1]
```diff
@@ -123,10 +132,10 @@ Anda dapat menyesuaikan — atau bahkan menonaktifkan — daftar isi secara glob
 
 Secara default, heading `<h2>` dan `<h3>` akan dimasukkan ke dalam daftar isi. Ubah tingkat heading yang akan dimasukkan di seluruh website menggunakan opsi `minHeadingLevel` dan `maxHeadingLevel` dalam konfigurasi [global `tableOfContents`](/id/reference/configuration/#tableofcontents). Timpa pengaturan default ini pada masing-masing halaman dengan menambahkan properti [frontmatter `tableOfContents`](/id/reference/frontmatter/#tableofcontents) yang sesuai.
 
-<Tabs>
+<Tabs syncKey="config-type">
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/customization.mdx:135]: Even tho this change is correct, the tabs are not synced as it requires identical `label`s. For the first set of tab, the second tab item label is `Konfigurasi global` and for the second one, it's `Konfigurasi Global` (notice the `g` / `G` difference). To get synced tabs, they both should be identical.

I'll let you pick the one that makes the most sense to you.

File: docs/src/content/docs/id/guides/customization.mdx [hunk 2]
```diff
@@ -171,7 +180,7 @@ tableOfContents: false
   </TabItem>
   <TabItem label="Konfigurasi Global">
 
-```js
+```js {7}
```
Comment (HiDeoo) [inline docs/src/content/docs/id/guides/customization.mdx:183]:
```suggestion
```js {8}
```

Due to formatting, the line with `tableOfContents` is 1 line below in this language.

File: docs/src/content/docs/id/reference/configuration.mdx [hunk 3]
```diff
@@ -346,6 +379,94 @@ starlight({
 });
 ```
 
+### `expressiveCode`
+
+**type:** `StarlightExpressiveCodeOptions | boolean`  
+**default:** `true`
+
+Starlight menggunakan [Kode Ekspresif](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) untuk merender blok kode dan menambahkan dukungan untuk menyorot bagian contoh kode, menambahkan nama file ke blok kode, dan banyak lagi.
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/configuration.mdx:387]: Regarding "Kode Ekspresif", Expressive Code is the name of the library and should not be translated (I think this should be fixed 5 times below if I'm correct).

File: docs/src/content/docs/id/reference/configuration.mdx [hunk 4]
```diff
@@ -346,6 +379,94 @@ starlight({
...
+
+#### `useStarlightDarkModeSwitch`
+
+**type:** `boolean`  
+**default:** `true`
+
+Jika `true`, blok kode secara otomatis beralih antara tema terang dan gelap saat tema situs berubah.
+Jika `false`, Anda harus menambahkan CSS secara manual untuk menangani peralihan antara beberapa tema.
+
+:::catatan
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/configuration.mdx:432]:
```suggestion
:::note
```

The type of asides should not be translated and should always be `note`, `tip`, `caution` or `danger`.

File: docs/src/content/docs/id/reference/configuration.mdx [hunk 5]
```diff
@@ -346,6 +379,94 @@ starlight({
...
+
+#### `useStarlightUiThemeColors`
+
+**type:** `boolean`  
+**default:** `true` jika `themes` tidak diatur, kalau tidak `false`
+
+Jika `true`, variabel CSS Starlight digunakan untuk warna elemen UI blok kode (latar belakang, tombol, bayangan, dll.), yang sesuai dengan [tema warna situs](/id/guides/css-and-tailwind/#tema).
+Jika `false`, warna yang disediakan oleh tema penyorotan sintaksis aktif digunakan untuk elemen-elemen ini.
+
+:::catatan
```
Comment (HiDeoo) [inline docs/src/content/docs/id/reference/configuration.mdx:444]:
```suggestion
:::note
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few comments and suggestions but otherwise, this looks good to me.

File: docs/src/content/docs/id/guides/customization.mdx (hunk 2 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/customization.mdx]: Ah i see, thank you for the feedback. I will fix it soon.

File: docs/src/content/docs/id/reference/configuration.mdx (hunk 3 above)
Comment (iyyafi) [reply docs/src/content/docs/id/reference/configuration.mdx]: Okay, will update all the "Expressive Code" translations immediately.

File: docs/src/content/docs/id/reference/configuration.mdx (hunk 4 above)
Comment (iyyafi) [reply docs/src/content/docs/id/reference/configuration.mdx]: Ah, i'm sorry. I'm not notice this was a code. I will fix it soon. Thank you.

File: docs/src/content/docs/id/reference/configuration.mdx (hunk 5 above)
Comment (iyyafi) [reply docs/src/content/docs/id/reference/configuration.mdx]: Will update this soon too.

File: docs/src/content/docs/id/guides/customization.mdx (hunk 1 above)
Comment (iyyafi) [reply docs/src/content/docs/id/guides/customization.mdx:135]: I see, will adjust so it will sync. Thank you.

**Review by HiDeoo — APPROVED** [review]
Thanks for the updates :raised_hands:

**Pull request merged**